src/twisted/python/test/test_release.py

```diff
@@ -1211,8 +1221,10 @@ def createFakeSphinxProject(self):
         files.  This includes a single source file ('index.rst') and the
         smallest 'conf.py' file possible in order to find that source file.
         """
-        self.sourceDir.child("conf.py").setContent(self.confContent)
-        self.sourceDir.child("index.rst").setContent(self.indexContent)
+        confPy = FilePathStrContent(self.sourceDir.child("conf.py"))
+        indexRst = FilePathStrContent(self.sourceDir.child("index.rst"))
+        confPy.setContent(self.confContent)
+        indexRst.setContent(self.indexContent)
```

Comment:
The coverage report says this whole function is un-executed.  Is that a codecov bug or does the CI system not exercise this code?